bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -263,7 +263,7 @@ data Resources
     , blockHeaderDb :: !BlockHeaderDb
     , pactService :: !(Async (), PactQueue)
     , mainTrunkBlocks :: ![T3 ParentHeader BlockHeader PayloadWithOutputs]
-    , coinAccounts :: !(MVar (Map Account (NonEmpty Ed25519KeyPairCaps)))
+    , coinAccounts :: !(MVar (Map Account (NonEmpty (DynKeyPair, [SigCapability]))))
```

Comment:
To avoid repetition, could we change this tuple into a proper sum type?

Reply:
Deferring to later, adding to the webauthn cleanup asana ticket.